broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java

```diff
@@ -94,16 +94,15 @@ public boolean isPopReviveServiceRunning() {
     @Override
     public RemotingCommand processRequest(final ChannelHandlerContext ctx,
         RemotingCommand request) throws RemotingCommandException {
-        return this.processRequest(ctx.channel(), request, true);
+        return this.processRequest(ctx.channel(), request);
     }
 
     @Override
     public boolean rejectRequest() {
         return false;
     }
 
-    private RemotingCommand processRequest(final Channel channel, RemotingCommand request,
-        boolean brokerAllowSuspend) throws RemotingCommandException {
+    private RemotingCommand processRequest(final Channel channel, RemotingCommand request) throws RemotingCommandException {
```

Comment:
This param may be useful in the future, so pls communicate with the community before modify it.